jbpm-human-task/jbpm-human-task-jpa/src/main/java/org/jbpm/services/task/impl/model/TaskImpl.java

```diff
@@ -65,6 +68,14 @@ public class TaskImpl implements InternalTask {
      * WSHT uses a name for the unique identifier, for now we use a generated ID which is also the key, which can be
      * mapped to the name or a unique name field added later.
      */
+	
+    private static final Logger logger = LoggerFactory.getLogger(TaskImpl.class);
+    
+    @Transient
+    private final int TASK_DESCRIPTION_LENGTH = Integer.parseInt(System.getProperty("org.jbpm.ht.task.description.length", "255"));
```

Comment:
```suggestion
    private static final int TASK_DESCRIPTION_LENGTH = Integer.parseInt(System.getProperty("org.jbpm.ht.task.description.length", "255"));
```

Reply:
Hi @afalhambra Values defined for system-property and DB column size should be in sync. We need to use Alter table query to modify DB column size based on value defined for system-property